### PR TITLE
Add note about `rejectOnNotFound` removal

### LINKED ISSUE #4999: Remove deprecated rejectOnNotFound from docs
main/content/400-reference/200-api-reference/050-prisma-client-reference.mdx

### PR DESCRIPTION
**Pull request by ruheni**
Closes #4999

### CODE REVIEW

**Review by janpio — DISMISSED** [review]
`rejectOnNotFound` is also mentioned in the query documentation and needs to be marked removed there as well.

Additionally, this is the absolute minimum we can do here. I would greatly appreciate if we would find a way to de-emphasize the `rejectOnNotFound` content (especially the global config one).

**Comment by ruheni** [comment]
I might be missing something, but I could only find it in the Reference after a global search. 🤔 

Could you point me to the page you're referring to?

**Comment by janpio** [comment]
There are more mentions of `rejectOnNotFound` further down:
- https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findunique
- https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#findfirst

You only added a note to the global usage of the feature in `PrismaClient` context.

**Comment by ruheni** [comment]
Thanks. I've made the update. 💚

> I just realized I'd made the update on the wrong branch but it should be reflected now

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 1]
```diff
@@ -462,12 +464,12 @@ Use model queries to perform CRUD operations on your models. See also: [CRUD](/c
...
+| `where`                         | `UserWhereUniqueInput`   | **Yes**  | Wraps all _unique_ fields of a model so that individual records can be selected. <br></br>From version 4.5.0, this type wraps all fields of a model. [Learn more](#filter-on-non-unique-fields-with-userwhereuniqueinput) |
+| `select`                        | `XOR<UserSelect, null>`  | No       | [Specifies which properties to include](/concepts/components/prisma-client/select-fields) on the returned object.                                                                                                         |
+| `include`                       | `XOR<UserInclude, null>` | No       | [Specifies which relations should be eagerly loaded](/concepts/components/prisma-client/relation-queries) on the returned object.                                                                                         |
+| `rejectOnNotFound` (deprecated) | `RejectOnNotFound`       | No       | If true, throw a `NotFoundError: No User found error`. <br></br>**Note:** `rejectOnNotFound`is deprecated in v4.0.0. From v4.0.0, use [`findUniqueOrThrow`](#finduniqueorthrow) instead.                                  |
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx]:
```suggestion
| `rejectOnNotFound` (deprecated) | `RejectOnNotFound`       | No       | If true, throw a `NotFoundError: No User found error`. <br></br>**Note:** `rejectOnNotFound` is deprecated in v4.0.0. From v4.0.0, use [`findUniqueOrThrow`](#finduniqueorthrow) instead.                                  |
```

Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx]: In the first column it still says "deprecated". It is now removed instead.
This is also still missing the information _when_ the functionality was removed.

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 2]
```diff
@@ -597,17 +599,17 @@ We introduced `findUniqueOrThrow` in v4.0.0. It replaces the [`rejectOnNotFound`
...
+| `skip`                          | `number`                                                     | No       | Specifies how many of the returned objects in the list should be skipped.                                                                                                                                                                  |
+| `take`                          | `number`                                                     | No       | Specifies how many objects should be returned in the list. When used with `findFirst`, `take` is implicitly `1` or `-1`. `findFirst` is only affected by whether the value is positive or negative - any negative value reverses the list. |
+| `rejectOnNotFound` (deprecated) | `RejectOnNotFound`                                           | No       | If true, throw a `NotFoundError: No User found error`. <br></br>**Note:** `rejectOnNotFound`is deprecated in v4.0.0. From v4.0.0, use [`findFirstOrThrow`](#findfirstorthrow) instead.                                                     |
```
Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx]:
```suggestion
| `rejectOnNotFound` (deprecated) | `RejectOnNotFound`                                           | No       | If true, throw a `NotFoundError: No User found error`. <br></br>**Note:** `rejectOnNotFound` is deprecated in v4.0.0. From v4.0.0, use [`findFirstOrThrow`](#findfirstorthrow) instead.                                                     |
```

Comment (janpio) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx]: See https://github.com/prisma/docs/pull/5018/files#r1259701051

**Review by janpio — COMMENTED** [review]
See comments

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**